layouts/partials/future.html

```diff
@@ -8,6 +8,7 @@
       {{- if ne ($.Scratch.Get "month") (dateFormat "January" .startdate ) -}}
         {{- $.Scratch.Set "month" (dateFormat "January" .startdate ) -}}
         {{- $.Scratch.Set "month-displayed" "false" -}}
+<br />
```

Comment:
I'd sorta rather do this with CSS; we can do some padding on the element itself rather than a `<br /> tag, but it's not the end of the world.